scripts/deploy.sh

```
@@ -0,0 +1,10 @@
+# bash here to have nvm setup, without it we will get "nvm: command not found"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)